pkg/ottl/ottlfuncs/func_unix_test.go

```diff
@@ -23,7 +23,7 @@ func Test_Unix(t *testing.T) {
 		{
 			name: "January 1, 2023",
 			seconds: &ottl.StandardIntGetter[any]{
-				Getter: func(ctx context.Context, tCtx any) (any, error) {
+				Getter: func(_ context.Context, _ any) (any, error) {
```

Comment:
This seems cleaner IMO. It should make the linter happy as well, right?
```suggestion
				Getter: func(context.Context, any) (any, error) {
```

Reply:
yes, this should make the linter happy as well, looks perfectly fine